### PR TITLE
Add ManaParameters::annual_decay_factor_percentage

### PR DESCRIPTION
**Pull request by thibault-martinez**
https://github.com/iotaledger/iota.go/pull/606
Waiting for a TIP update.

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/src/types/block/mana/parameters.rs [hunk 1]
```diff
@@ -21,11 +21,13 @@ use crate::types::block::{
 pub struct ManaParameters {
     /// The number of bits used to represent Mana.
     pub(crate) bits_count: u8,
-    /// The amount of potential Mana generated by 1 IOTA in 1 slot.
+    /// The amount of potential Mana generated by 1 microIOTA in 1 slot multiplied by 2^generation_rate_exponent.
     pub(crate) generation_rate: u8,
     /// The scaling of `mana_generation_rate` expressed as an exponent of 2.
+    /// The actual generation rate of Mana is given by generation_rate * 2^(-generation_rate_exponent).
```
Comment (Alex6323) [inline sdk/src/types/block/mana/parameters.rs:27]:
```suggestion
    ///
    /// The actual generation rate of Mana is given by generation_rate * 2^(-generation_rate_exponent).
```

File: sdk/src/types/block/mana/parameters.rs [hunk 2]
```diff
@@ -21,11 +21,13 @@ use crate::types::block::{
...
     pub(crate) generation_rate_exponent: u8,
     /// A lookup table of epoch index diff to mana decay factor.
+    /// The actual decay factor is given by decay_factors[epoch_diff] * 2^(-decay_factors_exponent).
```
Comment (Alex6323) [inline sdk/src/types/block/mana/parameters.rs:30]:
```suggestion
    ///
    /// The actual decay factor is given by decay_factors[epoch_diff] * 2^(-decay_factors_exponent).
```

File: sdk/src/types/block/mana/parameters.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/src/types/block/mana/parameters.rs:27]: Not of big fan of that tbh 🤷🏻

Comment (Alex6323) [reply sdk/src/types/block/mana/parameters.rs:27]: fair enough

**Comment by Alex6323** [comment]
some outdated `mana_` prefixes in the docs

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**